website_form_validations/README.rst

```diff
@@ -0,0 +1,7 @@
+Website Form Validations
```

Comment:
Use this template please.

https://github.com/OCA/maintainer-tools/tree/master/template/module